internal/pkg/views/list.go

```diff
@@ -78,7 +78,7 @@ func (w *ListWidget) SetFilter(filterString string) {
 	}
 	filteredItems := []*expanders.TreeNode{}
 	for _, item := range w.currentPage.Items {
-		if strings.Contains(strings.ToLower(item.Display), filterString) {
+		if strings.Contains(strings.ToLower(item.Display), strings.ToLower(filterString)) {
```

Comment:
Could you create `filterStringLower` outside the `for` loop to avoid performing the conversion multiple times?

Reply:
Good call. I've changed this instance, and the other instance where a filter is used in a loop